ui/src/main/webapp/app/graph/graph-service.js

```diff
@@ -38,13 +40,49 @@ angular.module('app').factory('graph', ['types', '$q', 'results', 'common', 'eve
         },
         physics: {
             springLength: 250,
+            gravity: -6,
+            theta: 0.95,
             fit: true
         },
         iterations: 10000,
         fit: true,
         animate: false
     };
 
+    var styling;
+
+    var defaultStyling = {
+        edges: {
```

Comment:
Could this default styling be moved out to config too? Then if a user just wants all edges to be blue, they could override the default? Also it is duplicated between the graph view and the schema view.